content/for-cisos/data-protection/gdpr.md

```diff
@@ -0,0 +1,16 @@
+
+# **GDPR**
+
+General Data Protection Regulation (GDPR) has been adopted by the European Union ("EU") and applies from May 25, 2018.
+
+1. Why is GDPR important?
+    GDPR establishes a stronger data protection regulatory framework for processing personal data of individuals. GDPR brings:
+
+    New and enhanced rights for individuals
+    Widened definition of personal data
+    New obligations for processors
+    Potential for significant financial penalties for non-compliance
+    Compulsory data breach notification
+2. Read more about GDPR
+    [EU GDPR Information Portal](https://www.eugdpr.org/)
```

Comment:
please add on separate lines or use bullets, they are not properly rendered now the links

depending on your code editor you can most often get markdown preview there

Reply:
updated